nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -687,8 +686,7 @@ public static Nullness getGenericParameterNullnessAtInvocation(
       return Nullness.NONNULL;
     }
     Type enclosingType =
-        castToNonNull(
-            getTreeType(((MemberSelectTree) tree.getMethodSelect()).getExpression(), state));
+        getTreeType(((MemberSelectTree) tree.getMethodSelect()).getExpression(), state);
```

Comment:
A bit confused about what we fixed here. Why would simply removing `castToNonNull` fix handling of casting to raw types? 🤔 to me it seems that we are just removing a (presumably now redundant) `castToNonNull`

Reply:
It's because in NullAway `castToNonNull` actually crashes the program.  Here the cast was unnecessary since `enclosingType` is passed to a method that accepts a `@Nullable` parameter